src/block-card.css

```diff
@@ -0,0 +1,76 @@
+.block-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  color: #060b35;
+  padding: 32px 16px 16px;
+}
+
+.block-card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.block-card__name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.block-card__id {
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.rating {
```

Comment:
each BEM-block styles should be written in a separate file. If 'rating' is a BEM-block, put it in a new file